tests/system/small/test_series.py

```diff
@@ -2565,6 +2565,30 @@ def test_between(scalars_df_index, scalars_pandas_df_index, left, right, inclusi
     )
 
 
+def test_case_when(scalars_df_index, scalars_pandas_df_index):
+    bf_series = scalars_df_index["int64_col"]
+    pd_series = scalars_pandas_df_index["int64_col"]
+
+    # TODO(tswast): pandas case_when appears to assume True when a value is
+    # null. I suspect this should be considered a bug in pandas.
+    bf_result = bf_series.case_when(
+        [
+            ((bf_series > 100).fillna(True), 1000),
+            ((bf_series < -100).fillna(True), -1000),
+        ]
+    ).to_pandas()
+    pd_result = pd_series.case_when(
```

Comment:
This isn't available until pandas 2.2, so we'll need a version based skip for this test.

Reply:
Done.